test/dummy/public/openapi.yaml

```diff
@@ -66,8 +66,43 @@ paths:
       responses:
         '204':
           description: ''
+  /incomplete_products:
+    post:
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ProductRequestParameter'
+      responses:
+        '201':
+          description: ''
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Product'
```

Comment:
No longer needed?

Reply:
It seems that "/incomplete_products" is found only here.

Reply:
> No longer needed?

Exactly...